compiler/luci/pass/include/luci/Pass/CanonicalizePass.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_CANONICALIZE_PASS_H__
+#define __LUCI_CANONICALIZE_PASS_H__
+
+#include <logo/Pass.h>
+
+namespace luci
+{
+
+/**
+ * @brief  Class to canoncalize CircleNodes
+ *
+ */
+struct CanonicalizePass final : public logo::Pass
```

Comment:
This pass name seems too general. This name more fits a stage, i.e., a sequence of passes.

How about renaming to CanonicalizePaddingDtypePass, or simply ConvertS64PaddingToS32Pass ?

Reply:
Named this pass to do some multiple things as canonicalization.
This pass will run always without option.